platform/email/src/main/java/org/motechproject/email/settings/SettingsDto.java

```diff
@@ -1,14 +1,22 @@
 package org.motechproject.email.settings;
 
+import org.codehaus.jackson.annotate.JsonIgnore;
 import org.motechproject.server.config.SettingsFacade;
+import org.springframework.util.CollectionUtils;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.Properties;
 
 public class SettingsDto {
+    public static final String EMAIL_ADDITIONAL_PROPERTIES_FILE_NAME = "motech-email-additional.json";
```

Comment:
What's the point of storing the additional properties file as json? I would prefer it a properties file